cynic-introspection/tests/snapshots/sdl_tests__spacex_sdl_conversion.snap

```diff
@@ -1147,4 +1148,37 @@ type _Service {
   sdl: String
 }
 
+directive @contact("""
+Contact title of the subgraph owner
+"""
+name: String!, """
+URL where the subgraph's owner can be reached
+"""
+url: String, """
+Other relevant notes can be included here; supports markdown links
+"""
+description: String) on SCHEMA
```

Comment:
This output is kind of trash, I should clean it up...

Possibly by updating cynic-introspection to use the cynic-parser pretty printer?